eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -23,23 +36,32 @@ stages:
                 - checkout: azure-sdk-build-tools
                   path: azure-sdk-build-tools
 
+                # 1es PT credscan step will fail if the suppression file path we use in other stages does not exist here.
+                # Checking out the repo via sparse checkout changes the devops path structure so that can't be used instead.
+                - pwsh: |
+                    $ErrorActionPreference = 'Stop'
+                    New-Item -ItemType Directory -Force eng
+                    $req = Invoke-WebRequest https://raw.githubusercontent.com/Azure/azure-sdk-for-net/main/eng/CredScanSuppression.json
+                    $req.RawContent | Out-File eng/CredScanSuppression.json
+                  workingDirectory: $(Pipeline.Workspace)/azure-sdk-build-tools
```

Comment:
I will need to steal this for `java` release phase for the same reason.

Reply:
Yeah basically anywhere we are checking out build tools in the release stages it's easier to do this. Otherwise if there's `checkout: none` you could use this OR do a parameter-less sparse checkout.